MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java

```diff
@@ -72,8 +76,12 @@
 	 * If unsure, leave unchanged.
 	 */
 	private final List<Function<Typo3Entry, Typo3Entry>> modifiers = Arrays.asList(
-			Modifiers.MARK_IF_THOMAS_IS_EDITOR
+			  Modifiers.MARK_IF_THOMAS_IS_EDITOR
```

Comment:
?

Reply:
I fixed the indent such that all modifiers are indented the same way disregarding commata. Maybe it happened even automatically. Like this:
```java
  Modifiers.MARK_IF_THOMAS_IS_EDITOR
, Modifiers.MARK_IF_TO_APPEAR
, Modifiers.ADD_PAPER_LINK_IF_SOFTVARE
```